BUG?: spatial_shuffle failure with the latest dask

```
...
dask_geopandas/tests/io/test_arrow.py::test_index FAILED                 [ 92%]
...

=================================== FAILURES ===================================
__________________________________ test_index __________________________________

tmp_path = PosixPath('/tmp/pytest-of-runner/pytest-0/test_index0')

    def test_index(tmp_path):
        # set up dataset
        df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
        # get meaningful index by shuffling (hilbert distance)
>       df = dask_geopandas.from_geopandas(df, npartitions=1).spatial_shuffle().compute()

dask_geopandas/tests/io/test_arrow.py:186: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
dask_geopandas/core.py:752: in spatial_shuffle
    sorted_ddf = self.set_index(
dask_geopandas/core.py:546: in set_index
    ddf = super().set_index(*args, **kwargs)
/usr/share/miniconda3/envs/test/lib/python3.9/site-packages/dask/dataframe/core.py:4700: in set_index
    return set_index(
/usr/share/miniconda3/envs/test/lib/python3.9/site-packages/dask/dataframe/shuffle.py:231: in set_index
    result = set_sorted_index(df, index, drop=drop, divisions=divisions)
/usr/share/miniconda3/envs/test/lib/python3.9/site-packages/dask/dataframe/shuffle.py:1105: in set_sorted_index
    result.divisions = tuple(divisions)
/usr/share/miniconda3/envs/test/lib/python3.9/site-packages/dask/dataframe/core.py:4444: in __setattr__
    object.__setattr__(self, key, value)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = Dask GeoDataFrame Structure:
              pop_est continent    name  iso_a3 gdp_md_est  geometry
npartitions=1       ...   float64  geometry
176               ...       ...     ...     ...        ...       ...
Dask Name: set_index, 6 tasks
value = (375036506, -530848116)

    @divisions.setter
    def divisions(self, value):
        if not isinstance(value, tuple):
            raise TypeError("divisions must be a tuple")
    
        if hasattr(self, "_divisions") and len(value) != len(self._divisions):
            n = len(self._divisions)
            raise ValueError(
                f"This dataframe has npartitions={n - 1}, divisions should be a "
                f"tuple of length={n}, got {len(value)}"
            )
    
        if None in value:
            if any(v is not None for v in value):
                raise ValueError(
                    "divisions may not contain a mix of None and non-None values"
                )
        else:
            # Known divisions, check monotonically increasing
    
            # XXX: if the index dtype is an ordered categorical dtype, then we skip the
            # sortedness check, since the order is dtype dependent
            index_dtype = getattr(self._meta, "index", self._meta).dtype
            if not (is_categorical_dtype(index_dtype) and index_dtype.ordered):
                if value != tuple(sorted(value)):
>                   raise ValueError("divisions must be sorted")
E                   ValueError: divisions must be sorted

/usr/share/miniconda3/envs/test/lib/python3.9/site-packages/dask/dataframe/core.py:410: ValueError
=============================== warnings summary ===============================
```

--- a/dask_geopandas/tests/io/test_arrow.py
+++ b/dask_geopandas/tests/io/test_arrow.py
@@ -183,7 +183,7 @@ def test_index(tmp_path):
     # set up dataset
     df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
     # get meaningful index by shuffling (hilbert distance)
-    df = dask_geopandas.from_geopandas(df, npartitions=1).spatial_shuffle().compute()
+    df = dask_geopandas.from_geopandas(df, npartitions=2).spatial_shuffle().compute()
     ddf = dask_geopandas.from_geopandas(df, npartitions=4)
 
     # roundtrip preserves the index by default

